resources/views/pages/claims/active.blade.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use App\Models\AchievementSetClaim;
+use App\Platform\Services\AchievementSetClaimListService;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . App\Models\AchievementSetClaim::class]);
```

Comment:
Opinion is less strong here, but I lean towards this should also be an unauthenticated route.